iree/samples/static_library/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+################################################################################
```

Comment:
this was not auto generated :)

Reply:
🤖  I'm a robot and I generate all the things.